app/views/curation_concerns/base/file_manager.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1><%= t("file_manager.link_text") %></h1>
```

Comment:
Is the same as the CC version?  I just eyeballed them and didn't see any difference.